docs/fundamentals/aspire-sdk-templates.md

```diff
@@ -0,0 +1,165 @@
+---
+title: .NET Aspire templates
+description: Learn how to install the .NET Aspire templates, and how to use them to create new apps.
+ms.date: 11/04/2024
+zone_pivot_groups: dev-environment
+uid: dotnet/aspire/templates
+---
+
+# .NET Aspire templates
+
+There are a number of .NET Aspire project templates available to you. You can use these templates to create full .NET Aspire solutions, or add individual projects to existing .NET Aspire solutions.
+
+The .NET Aspire templates are available in the [📦 Aspire.ProjectTemplates](https://www.nuget.org/packages/Aspire.ProjectTemplates) NuGet package.
+
+## Install the .NET Aspire templates
+
+:::zone pivot="visual-studio"
+
+The .NET Aspire templates are installed automatically when you install Visual Studio 17.9 or later.
```

Comment:
As you and I chatted offline, this is not actually true. You can use aspire with 17.9 and above, but it is not guaranteed to have the templates automatically installed. The only version that comes with the templates installed built-in (and note that this is only the case if you check the web workload) is 17.12 GA.

Reply:
The other thing that we talked about, is that for users that want to still use Aspire 8.x, they will now have to make sure they check the checkbox in the installer so that the Aspire workload is installed. Not sure if that is something we want to have in docs or not, specially given that when we ship 9.0 8.x will be out of support, but just mentioning it in case you think we should.

Reply:
Not sure if this is too verbose

```suggestion
To install the .NET Aspire templates in Visual Studio, you need to manually install them unless you are using Visual Studio 17.12 or later. For Visual Studio 17.9 to 17.11, follow these steps:

1. Open Visual Studio.
2. Go to **Tools** > **NuGet Package Manager** > **Package Manager Console**.
3. Run the following command to install the templates:

  ```powershell
  dotnet new install Aspire.ProjectTemplates
  `` ****(add an extra backtick here. I ommited it so suggestion wouldn't look cropped.)****

For Visual Studio 17.12 or later, the .NET Aspire templates are installed automatically.
```